systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -178,10 +183,11 @@ void testTieredStorageWithAivenPlugin() {
                 return earliestLocalOffset > 0;
             });
 
-        ClientUtils.waitForInstantProducerClientSuccess(testStorage);
```

Comment:
Since we have produced 10000 messages in L156 above, we don't need to produce messages again here.